apps/researcher/src/app/[locale]/objects/[id]/metadata.tsx

```diff
@@ -39,48 +37,48 @@ export function MetadataContainer({identifier, children}: Props) {
 
 interface MetadataEntryProps {
   children: ReactNode;
-  isOwner: boolean;
+  isCurrentLocation: boolean;
 }
 
-export function MetadataEntry({children, isOwner = false}: MetadataEntryProps) {
-  const {name, identifier} = useCurrentOwner.getState();
+export function MetadataEntry({
+  children,
+  isCurrentLocation = false,
+}: MetadataEntryProps) {
+  const {identifier} = useMetadata.getState();
+  const organization = useCurrentPublisher.getState();
   const t = useTranslations('ObjectDetails');
 
   if (!children) {
     return (
-      <div className="w-full flex flex-col gap-2">
-        <div className="text-neutral-600 italic w-full border-t p-6">
-          {t.rich('noData', {
-            subject: t(identifier),
-            subjectWrapper: chunks => (
-              <span className="lowercase">{chunks}</span>
-            ),
-          })}
-        </div>
+      <div className="text-neutral-600 italic w-full border-t py-6 text-sm">
+        {t.rich('noData', {
+          subject: () => <span className="lowercase">{t(identifier)}</span>,
+        })}
       </div>
     );
   }
 
   return (
-    <div className="border border-neutral-200 text-neutral-900 rounded flex flex-col lg:flex-row justify-between gap-2 lg:divide-x divide-sand-100">
-      <div className="w-full lg:w-3/5 px-2 py-3 ">{children}</div>
-      <div className="w-full lg:w-2/5 px-2 py-3 text-xs text-sand-700 mt-1 flex flex-row justify-between gap-2 ">
+    <div className="border-neutral-200 text-neutral-900 rounded border-t flex flex-col lg:flex-row justify-between gap-2 lg:divide-x divide-sand-100">
+      <div className="w-full lg:w-3/5  py-3">{children}</div>
+      <div className="w-full lg:w-2/5 px-2 py-3 text-xs text-sand-700 mt-1 flex flex-row justify-between gap-2">
         <div className="flex flex-col gap-1">
           <div>
-            {t('addedBy')}
-            <strong className="ml-1">{name}</strong>
-            {/* Form <em>Amsterdam, Netherlands</em>
-            on <em>12-12-2018</em> */}
-          </div>
-          {isOwner && <div>{t('currentOwner')}</div>}
-        </div>
-        <div className="flex shrink">
-          <div>
-            <button className="flex items-center py-2 px-3 rounded-full bg-sand-100 text-sand-900 hover:bg-white transition text-xs">
-              <HandThumbUpIcon className="w-4 h-4 stroke-sand-600" />
-              &nbsp;(0)
-            </button>
+            {t.rich('addedBy', {
```

Comment:
Minor - not something to resolve now, but to think about: the data doesn't say that the data publisher has 'added' this information - it could have been added by another organization. But we do now that the information has been 'provided' by the publisher.

Reply:
Good point. Maybe also something to think about in the design. Now the text in the design is:

![image](https://github.com/colonial-heritage/colonial-collections/assets/1481602/318f1a5b-7291-40af-8a29-7f1a43fbe9cc)